libbeat/docs/security/users.asciidoc

```diff
@@ -185,71 +185,68 @@ endif::serverless[]
 [[privileges-to-publish-events]]
 ==== Grant privileges and roles needed for publishing
 
-Users who publish events to {es} need to create and read from {beatname_uc}
+Users who publish events to {es} need to create and write to {beatname_uc}
 indices. To minimize the privileges required by the writer role, you can use the
-<<privileges-to-setup-beats,setup role>> to pre-load dependencies. Then turn off
-setup options in the  {beatname_uc} config file before running {beatname_uc} to
-publish events. For example:
+<<privileges-to-setup-beats,setup role>> to pre-load dependencies.
 
 ifndef::no_ilm[]
 [source,yaml]
-----
-setup.template.enabled: false
-setup.ilm.check_exists: false
-setup.ilm.overwrite: false <1>
-----
-<1> Omit `ilm.check_exists` and `ilm.overwrite` if ILM is disabled.
-endif::no_ilm[]
+When using ILM, turn off the ILM setup check in the {beatname_uc} config file before
+running {beatname_uc} to publish events:
 
-ifdef::no_ilm[]
-[source,yaml]
 ----
-setup.template.enabled: false
+setup.ilm.check_exists: false
 ----
 endif::no_ilm[]
 
 To grant the required privileges:
 
-. Create a *writer role*, called something like +{beat_default_index_prefix}_writer+, that has
-the following privileges (this list assumes the setup options shown earlier are
-set to `false`):
+. Create a *writer role*, called something like +{beat_default_index_prefix}_writer+,
+that has the following privileges:
+
+NOTE: Only the `monitor` cluster privilege and the `create` index privilege on
++{beat_default_index_prefix}-*+ indices are required in every configuration.
+
 +
 [options="header"]
 |====
 |Privileges | Why needed?
 
 ifndef::apm-server[]
 |`monitor`
-|Send monitoring info
+|Retrieve cluster details (e.g. version)
 endif::apm-server[]
 
-ifndef::no_ilm[]
-|`read_ilm`
-|Read the ILM policy when connecting to clusters that support ILM
-endif::no_ilm[]
+|`create` on +{beat_default_index_prefix}-*+ indices
+|Index events into {es}
 
 ifeval::["{beatname_lc}"=="filebeat"]
-|`manage_pipeline`
-|Load ingest pipelines used by modules
+|`cluster:admin/ingest/pipeline/get`
```

Comment:
> I also had to grant `cluster:admin/ingest/pipeline/get`

Yes, that makes sense, but you didn't need `manage_pipeline`, right? That's why I changed it to `cluster:admin/ingest/pipeline/get`.

Reply:
Everything worked fine when I added `cluster:admin/ingest/pipeline/get`. I just realized that I only run through the steps with ILM enabled, tho. I should probably check with daily indices.

Reply:
@cwurm I somehow managed to say `get` instead of `put` in both of my comments here ^^. For this to work, I had to add `cluster:admin/ingest/pipeline/put`.

Running out to buy a new pair of reading glasses..... :-P

Reply:
It should work if you run `./filebeat module enable system` followed by `./filebeat setup` (with setup privileges). Setup should install the pipelines, then plain `./filebeat` should just run.

Reply:
Aha...I must have run the setup before enabling the correct module. It works now.